htdocs/societe/class/societe.class.php

```diff
@@ -1553,6 +1553,18 @@ public function delete($id, User $fuser = null, $call_trigger = 1)
 				}
 			}
 
+			// Remove third party  TODO trigger for external action linked to these accounts
```

Comment:
Why adding this code ? There exactly the same just after...

Reply:
@eldy bad paste.. i fixed it... so many phone call...